.github/workflows/rust-audit.yml

```
@@ -0,0 +1,20 @@
+name: rust-audit
+permissions:
+  issues: write
+on:
+  schedule:
+    # Run at 00:00 UTC every day
+    - cron: '0 0 * * *'
+jobs:
```

stylistic nit, but our other workflows specify the `on` before `permisisons` and use more whitespace. also there's no need to make the name match the filename

```suggestion
name: RUSTSEC audit

on:
  schedule:
    # Run at 00:00 UTC every day
    - cron: '0 0 * * *'

permissions:
  issues: write

jobs:
```